.tekton/osc-operator-pull-request.yaml

```diff
@@ -14,7 +14,7 @@ metadata:
       files.all.exists(path, !path.matches('must-gather/.*|.tekton/.*must-gather.*yaml$')) &&
       files.all.exists(path, !path.matches('config/peerpods/podvm/.*|.tekton/.*podvm-builder.*yaml$')) &&
       files.all.exists(path, !path.matches('.tekton/.*fbc-.*|fbc/.*')) &&
-      files.all.exists(path, !path.matches('config/manager/.*'))
+      files.all.exists(path, !path.matches('config/manager/manager.yaml'))
```

Comment:
What about moving the `!` like below?

```suggestion
      !files.all.exists(path, path.matches('config/manager/.*'))
```

This should provide more predictable outputs. I tested it in [CEL Playground](https://playcel.undistro.io/).

Reply:
yes, I'd do the same on all of them